chunky/src/java/se/llbit/chunky/resources/Image.java

```diff
@@ -0,0 +1,31 @@
+package se.llbit.chunky.resources;
+
+/**
+ * Generic image giving read only access
+ */
+public interface Image {
+  /**
+   * Get the width of the image
+   */
+  int getWidth();
+
+  /**
+   * Get the height of the image
+   */
+  int getHeight();
+
+  /**
+   * Get the pixel at position (x, y)
+   */
+  int getPixel(int x, int y);
+
+  /**
+   * Get the pixel at the given index. Index is computed as y * width + x
+   */
+  int getPixel(int index);
```

Comment:
Should we drop this method from the `Image` interface? The usages of it could as well use the `getPixel(x, y)`, which would even simplify the code.